Sources/Proxy/Representations/SliceRepresentationProxy/index.js

```diff
@@ -255,10 +255,12 @@ function vtkSliceRepresentationProxy(publicAPI, model) {
     const bds = ds.getBounds();
     const axisIndex = 'XYZ'.indexOf(slicingMode);
     const endValue = bds[axisIndex * 2 + 1];
+    const ijk = model.mapper.getClosestIJKAxis().ijkMode;
+    const spacing = ds.getSpacing()[ijk];
     let currentValue = bds[axisIndex * 2];
     while (currentValue <= endValue) {
       values.push(currentValue);
-      currentValue++;
+      currentValue += spacing;
```

Comment:
Can the proxy be in IJK mode? If it's in IJK mode, the values should iterate by 1. Not sure that `SliceRepresentationProxy` supports it, or only XYZ.

Reply:
Oh you're right, SliceRep does support IJK, but View2DProxy does not. I've been looking at this from the perspective of View2DProxy, but SliceRep does support IJK.

Reply:
There's actually a lot of code in SliceRep that assumes XYZ and not IJK, so changing things would be more involved...

Reply:
Yeah, I actually have two projects that use this in IJK mode when looking back.

What is the goal of `getSliceValues` when we have `updateDomains` already defining all the necessary values? See [here](https://github.com/Kitware/vtk-js/commit/7182b5a16c778ba92e788d61814d56113969a3ee) from the commit two years ago.

Reply:
I don't actually know why we have getSlicingValues(). I assumed it was to handle non-uniform spacing, but the logic wasn't even there for that. I'll look into using the results of updateDomains in Glance, rather than mess with getSlicingValues().

Reply:
Thanks @floryst. Looks like the result of `updateDomains` is cached in `state`.

Reply:
Looking at your commit @agirault , you renamed `getSliceIndexValues` by `getSliceValues`. So you replaced an `index` based method to a generic one when the domain was supposed to take care of the generic path. So now, indeed, the `getSliceValues` is useless and should probably be removed.

Reply:
My bet is that `getSliceIndexValues()` was pre-dating the domain part.

Reply:
Thanks @jourdain.

I guess `getSliceValues` (and the previous `getSliceIndexValues`) could be useful if you don't only want the min, max, and step, but want all the values?

It's ironic that I linked to the commit where I made that change and did not properly support IJK at the time. Apologies for my 2y old mistake 🙏I was probably only using `updateDomains` and did not need all the slice values.